app/bundles/PageBundle/Controller/PageController.php

```diff
@@ -113,8 +113,14 @@ public function indexAction($page = 1)
             $filter['force'][] = ['column' => 'p.translationParent', 'expr' => 'isNull'];
         }
 
-        $orderBy    = $this->get('session')->get('mautic.page.orderby', 'p.title');
-        $orderByDir = $this->get('session')->get('mautic.page.orderbydir', 'DESC');
+        $orderBy    = $this->get('session')->get('mautic.page.orderby', 'p.dateModified');
+
+        $orderByDir = $this->get('session')->get('mautic.page.orderbydir');
+        if (!$orderByDir) {
+            $orderByDir = 'DESC';
+            $this->get('session')->set('mautic.page.orderbydir', $orderByDir);
+        }
```

Comment:
These lines do not need, do we?

Reply:
we need these lines because if we don't set session to DESC, if we click on modified date order the order will be remains to DESC, this because  setListFilters of CommonController set order to DESC if order doesn't exists in session.

These are the lines of code:
``` 
 if ($this->request->query->has('orderby')) {
       $orderBy = InputHelper::clean($this->request->query->get('orderby'), true);
       $dir     = $session->get("$name.orderbydir", 'ASC');
       $dir     = ('ASC' == $dir) ? 'DESC' : 'ASC';
       $session->set("$name.orderby", $orderBy);
       $session->set("$name.orderbydir", $dir);
   }
```

Reply:
I did test without these lines and it works.
I would like push the changes, but it return permission issue.

Reply:
Have you tried to enter go into the list and click on modify date order? However i pushed the changes

Reply:
I have solution for it.  

Can you merge actual 4.x branch, because it wrote:

> This branch is out-of-date with the base branch

Reply:
Done

Reply:
I saw the solution, it seem's a good solution

Reply:
I still see 

> This branch is out-of-date with the base branch

Reply:
Done again